src/components/filters/filter_menu/filter_menu.xml

```diff
@@ -3,22 +3,26 @@
     <div class="o-filter-menu d-flex flex-column bg-white" t-on-wheel.stop="">
       <t t-if="isSortable">
         <div>
-          <div class="o-filter-menu-item" t-on-click="() => this.sortFilterZone('ascending')">
+          <div
+            class="o-filter-menu-item my-2 mb-3"
+            t-on-click="() => this.sortFilterZone('ascending')">
             Sort ascending (A ⟶ Z)
           </div>
-          <div class="o-filter-menu-item" t-on-click="() => this.sortFilterZone('descending')">
+          <div class="o-filter-menu-item my-2" t-on-click="() => this.sortFilterZone('descending')">
             Sort descending (Z ⟶ A)
           </div>
         </div>
         <div class="o-separator"/>
       </t>
       <div class="o-filter-menu-actions">
-        <div class="o-filter-menu-action-text" t-on-click="selectAll">Select all</div>
-        <div class="o-filter-menu-action-text" t-on-click="clearAll">Clear</div>
+        <div class="o-button-link me-2 text-decoration-underline" t-on-click="selectAll">
```

Comment:
o-button-link are not underlined in the side panel. Idk if it fits here or not

Reply:
I agree that underline isn't the prettiest. But without the underline, you see "Select all Clear" without any clear separation between the two buttons. I didn't want to completely reinvent how it looked. I'll add a not to François if he wants to improve it.